src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private Random randomColor;
+    private int index;
+
+    public int getIndex() {
+        return index;
+    }
+
+    public void setIndex(int index) {
+        this.index = index;
+    }
+
+    public void setRandomColor(Random randomColor) {
+        this.randomColor = randomColor;
+    }
```

Comment:
```suggestion
```

Reply:
I can't understand what should I do in this case?